src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -327,7 +324,7 @@ ApiGlobe.prototype.createSceneGlobe = function createSceneGlobe(coordCarto, view
 
     var coordinate = new C.EPSG_4326(coordCarto.longitude, coordCarto.latitude, coordCarto.altitude);
 
-    this.scene = Scene(coordinate, viewerDiv, debugMode, gLDebug);
+    this.scene = Scene('EPSG:4978', coordinate, viewerDiv, debugMode, gLDebug);
```

Comment:
Warning, bad WGS84 axes, see #246, Y is up axis in iTowns and WGS84 Z is the up axis
Maybe add a comment to wait fix in #246
As you pay attention to the order of the axes?